android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandler.kt

```diff
@@ -26,13 +26,16 @@ class LogtoWebViewSocialHandler(
         window.logtoNativeSdk = {
             platform: 'android',
             getPostMessage: () => (data) => window.$NAME.postMessage(JSON.stringify(data)),
-            supportedSocialConnectorTargets: [${getSupportedSocialConnectorTargets()}],
+            supportedConnector: {
+                universal: true,
```

Comment:
Per offline discussion, not sure whether we should add a comment here for further explanation on setting universal to be true.

Reply:
Added.